content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -287,6 +287,35 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl on macOS
+
+Depending on how you installed ```kubectl```, use one of the following methods.
+
+### Uninstall kubectl using the command-line
+
+1. Locate the ```kubectl``` binary on your system:
+
+```bash
+where kubectl
+```
+
+2. Remove the ```kubectl``` binary:
+
+```bash
+sudo rm <path>
+```
+Replace with the path to the ```kubectl``` binary from the previous step. For example, ```sudo rm /usr/local/bin/kubectl```
```

Comment:
```suggestion
Replace `<path>` with the path to the `kubectl` binary from the previous step. For example, `sudo rm /usr/local/bin/kubectl`.
```

Reply:
Single backticks plus telling them which thing to replace (I missed it in my last suggestion)